ush/python/pyufs/ufswm/ufswm.py

```diff
@@ -0,0 +1,114 @@
+"""
+Module
+------
+
+    ufswm.py
+
+Description
+-----------
+
+    This module contains the base-class object for all Unified
+    Forecast System (UFS) weather-model (WM) applications.
+
+Classes
+-------
+
+    UFSWM(config, model, attr, *args, **kwargs)
+
+        This is the base-class object for all UFS weather-model
+        forecast and related applications; it is a sub-class of Task.
+
+Author(s)
+---------
+
+
+    Henry R. Winterbottom; 28 March 2023
+
+History
+-------
+
+    2023-03-28: Henry Winterbottom -- Initial implementation.
+
+"""
+
+# ----
+
+from pygw.logger import Logger
+from pygw.task import Task
+from pyufs.exceptions import UFSWMError
+
+# ----
+
+
+class UFSWM(Task):
```

Comment:
This should not be a subclass of `Task`.  
This should be considered as the base class of the ufs-weather-model.

Reply:
I am following this: https://github.com/NOAA-EMC/global-workflow/blob/develop/ush/python/pygfs/task/analysis.py#L17

What am I misunderstanding?